rust_src/src/frames.rs

```diff
@@ -97,6 +152,13 @@ impl LispFrameRef {
     }
 }
 
+impl Default for LispFrameRef {
+    #[inline]
+    fn default() -> LispFrameRef {
+        return selected_frame().as_frame_or_error();
```

Comment:
`selected_frame` returns a `LispObject`. this is just for convenience so i can call `unwrap_or_default` on a `Option<LispFrameRef>`. looking at the source for `as_frame_or_error`, it doesn't look like this adds anymore overhead than a simple `unwrap`, except there's no chance for a panic.

Reply:
@Wilfred this is an internal API we could clean up. selected_frame() should return a usable frame. However, sometimes the next routine called needs the LispObject so we would be LispObject -> frame -> LispObject which is kind of silly. It might make sense as the port continues to have a selected_frame_as_obj() and have selected_frame return a frame. For now, as @covercash2 says the call is safe and won't fail so there is little overhead. If we start seeing performance issues we can behave more like the C code and use XFOO style functions.